models/fivetran_platform__audit_table.sql

```diff
@@ -6,7 +6,7 @@
         'data_type': 'timestamp',
         'granularity': 'day'
     } if target.type == 'bigquery' else none,
-    incremental_strategy = 'merge' if target.type not in ['postgres', 'redshift'] else 'delete+insert',
+    incremental_strategy='insert_overwrite' if target.type in ('bigquery', 'spark', 'databricks') else 'delete+insert',
     file_format = 'delta'
```

Comment:
ah right i'll update

also should i add a partition field for databricks? currently we only have it for bigquery, partly because the dbt-bigquery adapter can accept the `sync_start` timestamp field and adjust its date granularity. for databricks, we'd have to add a field that is a `date_trunc('day')` of `sync_start` to the model itself (we could create this field in an `{% if target.type in ('spark', 'databricks') %}` block however so other users don't have an unnecessary field ...)